main/res/values-de/strings.xml

```diff
@@ -19,24 +19,24 @@
   <string name="mystery">Mystery-Cache</string>
   <string name="letterbox">Letterbox Hybrid</string>
   <string name="event">Event-Cache</string>
-  <string name="mega">Mega Event-Cache</string>
-  <string name="giga">Giga-Event Cache</string>
+  <string name="mega">Mega-Event-Cache</string>
+  <string name="giga">Giga-Event-Cache</string>
   <string name="earth">Earth-Cache</string>
-  <string name="cito">Cache in Trash out Event</string>
+  <string name="cito">Cache-In-Trash-Out-Event</string>
   <string name="webcam">Webcam-Cache</string>
   <string name="virtual">Virtual-Cache</string>
   <string name="wherigo">Wherigo-Cache</string>
   <string name="lostfound">Lost &amp; Found Event Cache</string>
-  <string name="ape">Project Ape-Cache</string>
+  <string name="ape">Project-APE-Cache</string>
   <string name="gchq">Groundspeak HQ</string>
   <string name="gps">GPS Adventures Exhibit</string>
   <string name="block">Groundspeak Block Party</string>
   <string name="unknown">Unbekannter Typ</string>
   <!-- cache sizes -->
-  <string name="cache_size_micro">Micro</string>
-  <string name="cache_size_small">Small</string>
-  <string name="cache_size_regular">Regular</string>
-  <string name="cache_size_large">Large</string>
+  <string name="cache_size_micro">Mikro</string>
+  <string name="cache_size_small">Klein</string>
+  <string name="cache_size_regular">Normal</string>
+  <string name="cache_size_large">Groß</string>
```

Comment:
Kann man die Diskussion irgendwo nachlesen? Eine Zielgruppe, denen die deutsche Version helfen würde, sind z. B. Kinder, die zwar lesen können, aber noch keinen Englisch-Unterricht haben (in BaWü wären das Grundschulkinder).